trashtalk/apps/cleanups/models.py

```diff
@@ -7,6 +7,35 @@
 from .utils import Coordinates
 
 
+# pylint: disable=missing-docstring
+class ToolCategory(models.Model):
+    name = models.CharField(max_length=128, unique=True)
+    description = models.TextField(blank=True)
+
+    class Meta:
+        verbose_name_plural = "tool categories"
+
+    def __str__(self):
+        return self.name
+
+
+# pylint: disable=missing-docstring
+class Tool(models.Model): # pylint: disable=too-few-public-methods
+    name = models.CharField(max_length=128)
+    description = models.TextField(blank=True)
+    image_static_location = models.CharField(max_length=200, blank=True)
```

Comment:
`image` is sufficient here. I assume we only need the image name and extension since we want to serve the images from the assets?

Whats the reason for `is_available`?

Reply:
re: the image field, yes that deserves some more discussion. Let's revisit the next time we chat in person.

Reply:
Please keep conversation going until such time. Not necessary to wait.